### PR TITLE
chore(web): Nv 3810 remove information architecture ff

### PR DESCRIPTION
**Pull request by SokratisVidros**
### What changed? Why was the change needed?

Remove the legacy feature flag and clean up the branching logic in the web app after the latest release.

### CODE REVIEW

File: apps/web/src/components/nav/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { useLocation } from 'react-router-dom';
```
Comment (SokratisVidros) [inline apps/web/src/components/nav/Sidebar.tsx:1]: This is a revised Sidebar component for the post IA era.

File: apps/web/src/constants/routes.ts [hunk 2]
```diff
@@ -42,7 +42,9 @@ export enum ROUTES {
   SETTINGS = '/settings',
   PROFILE = '/settings/profile',
   TEAM_SETTINGS = '/settings/team',
-  BRAND_SETTINGS = '/settings/brand',
+  /* Deprecated */
+  BRAND_SETTINGS_DEPRECATED = '/settings/brand',
```
Comment (SokratisVidros) [inline apps/web/src/constants/routes.ts:46]: This is just to align URLs with the copywriting on the sidebar.

File: apps/web/src/components/docs/docs.const.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { ROUTES } from '../../constants/routes';
 
 export const PATHS: { [key in ROUTES]?: string } = {
   [ROUTES.ACTIVITIES]: 'activity-feed/introduction',
+  // TODO: Where is this endpoint after IA?
   [`${ROUTES.BRAND}/layouts`]: 'content-creation-design/layouts',
```
Comment (SokratisVidros) [inline apps/web/src/components/docs/docs.const.ts]: @antonjoel82 @BiswaViraj is `/brand/layout` page still around post IA? My understanding is that it's not but I want to double-check with you.

Comment (davidsoderberg) [reply apps/web/src/components/docs/docs.const.ts]: This should be removable now as the mapping on the row below it should be the new router for layouts.

File: apps/web/src/constants/routes.ts [hunk 4]
```diff
@@ -42,7 +42,9 @@ export enum ROUTES {
   SETTINGS = '/settings',
   PROFILE = '/settings/profile',
   TEAM_SETTINGS = '/settings/team',
-  BRAND_SETTINGS = '/settings/brand',
+  /* Deprecated */
```
Comment (antonjoel82) [inline apps/web/src/constants/routes.ts:45]:
```suggestion
  /* @deprecated */
```

File: apps/web/src/components/nav/Sidebar.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+export const Sidebar = () => {
+  const { pathname } = useLocation();
+
+  let Variant = RootNavMenu;
+
+  if (pathname.startsWith(ROUTES.SETTINGS)) {
+    Variant = SettingsNavMenu;
+  } else if (pathname.startsWith(ROUTES.STUDIO)) {
+    Variant = LocalNavMenu;
+  }
```
Comment (antonjoel82) [inline apps/web/src/components/nav/Sidebar.tsx:30]: 💭 **thought (non-blocking):** I'm not sure that this revision is an improvement over the original and will just add verbosity through a chain of `if / else if`‏

File: apps/web/src/pages/settings/SettingsPage.tsx
Comment (antonjoel82) [inline apps/web/src/pages/settings/SettingsPage.tsx:1]: I believe this whole page can be deleted since it's the old, tabs-based settings page

**Review by antonjoel82 — APPROVED** [review]
Looks good, thanks!

File: apps/web/src/pages/settings/SettingsPage.tsx
Comment (SokratisVidros) [reply apps/web/src/pages/settings/SettingsPage.tsx:1]: Done

Comment (SokratisVidros) [reply apps/web/src/pages/settings/SettingsPage.tsx:1]: Removed

File: apps/web/src/components/nav/Sidebar.tsx (hunk 5 above)
Comment (SokratisVidros) [reply apps/web/src/components/nav/Sidebar.tsx:30]: I hear you. I considered adding a local Route component to handle the complicity but it felt too much for now. Settings will change very soon and I don't foresee more nesting in our sidebar but it's definitely something we need to refactor if sidebar becomes more complicated.

**Pull request merged**